api/gateway/share_test.go

```diff
@@ -137,6 +60,8 @@ var sharesBase64JSON string
 //
 // https://github.com/celestiaorg/celestia-app/issues/1816
 func Test_dataFromSharesBSR(t *testing.T) {
+	t.Skip("skip until sharesBase64JSON is regenerated with v1 compatibility")
```

Comment:
@rootulp, I don't know how to regenerate and where to get similar types of data, os skipping this for now

Reply:
https://github.com/celestiaorg/celestia-node/issues/2267